modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3_{context}"]
+= Release notes for {gitops-title} 1.3
+
+{gitops-title} 1.3 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="new-features-1-3_{context}"]
+== New features
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.0:
+
+* After the  Operator is installed, the Red Hat Single Sign On (SSO) is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Red Hat SSO installation after the Operator is installed which will result in removing the Red Hat SSO deployment from the `openshift-gitops` namespace.
```

Comment:
This statement is not true. Please replace `Red Hat Single Sign on` with `Dex`. It should look something like this.

For a fresh install of v1.3.0, Dex is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Dex installation after the Operator is installed which will result in removing the Dex deployment from the `openshift-gitops` namespace.

Reply:
We should mention that they can use `kubeadmin` credentials with dex as well, right?

Reply:
@iam-veeramalla and @jaideepr97 I have made the changes. Please check and confirm. Need your LGTM before merging